spyder/plugins/remoteclient/widgets/connectiondialog.py

```diff
@@ -426,28 +424,127 @@ def _create_configfile_subpage(self):
         configfile = self.create_browsefile(
             text=_("Configuration file *"),
             option=f"{self.host_id}/configfile",
+            tip=_("File with the OpenSSH client configuration to use"),
             alignment=Qt.Vertical,
             status_icon=ima.icon("error"),
         )
 
+        host = self.create_lineedit(
+            text=_("Host *"),
+            option=f"{self.host_id}/{AuthenticationMethod.ConfigFile}/address",
+            default="",
+            tip=_(
+                "This is the host (<i><b>Host</b></i> configuration keyword) "
```

Comment:
```suggestion
                "This is the host (i.e. <b>Host</b> configuration keyword) "
```

I tried to improve the text in parenthesis a bit but I don't understand very well what you mean by it. Could you explain why it's necessary?

Reply:
The value that needs to be given here refers to one of the `Host` values that is specified on the config file selected ( https://man.openbsd.org/ssh_config.5#Host). I thought that adding some sort of note for this could be important since the `Host` value can differ from the actual `Hostname` (https://man.openbsd.org/ssh_config.5#Hostname). As an example, if you have a config like:

```
Host servernickname
  Hostname 127.0.0.1
```

You will need to input `servernickname`. Putting `127.0.0.1` will make the connection fail (as far as I remember when I was checking this)